src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -179,20 +183,31 @@ class JesBackend extends Backend with LazyLogging {
       taskOutput.name -> rawValue
     }.toMap
 
-    status match {
-      case Run.Success(created, started, finished) =>
-        val taskOutputEvaluationFailures = outputMappings.filter {_._2.isFailure}
-        if (taskOutputEvaluationFailures.isEmpty) {
-          val unwrappedMap = outputMappings.collect { case (name, Success(wdlValue) ) => name -> wdlValue }
-          Success(unwrappedMap)
-        } else {
-          val message = taskOutputEvaluationFailures.collect { case (name, Failure(e))  => s"$name: $e" }.mkString("\n")
-          Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: $message"))
-        }
-      case Run.Failed(created, started, finished, errorCode, errorMessage) =>
-        Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: errorCode $errorCode for command: $commandLine. Message: $errorMessage"))
+    val warnAboutStderrLength: BigInteger = if(!call.failOnStderr) {
```

Comment:
space after `if`

It's a judgement call but if a branch is short/one line generally people pull it up on the same line without the braces